lib/Entity/Token.php

```diff
@@ -49,11 +44,15 @@ public function getTokenData()
     }
 
     /**
+     * This method is kept for BC and will be removed in the future.
+     *
+     * @deprecated
+     *
      * @return int
      */
     public function getVideoManagerId()
     {
-        return $this->videoManagerId;
+        return null;
```

Comment:
What kind of BC when it won't work at all? :D
I think it's better to bump one major version instead.

Reply:
Fair point. I can do that.